config/config-api/src/com/thoughtworks/go/util/command/EnvironmentVariableContext.java

```diff
@@ -227,25 +224,21 @@ public int hashCode() {
         return properties != null ? properties.hashCode() : 0;
     }
 
-    public String reportText() {
-        throw new RuntimeException("IMPLEMENT ME");
-    }
-
-    public void setupRuntimeEnvironment(Map<String, String> env, ConsoleOutputStreamConsumer consumer) {
+    public List<String> report() {
```

Comment:
I will just keep the old behavior. We can easily switch to not consider process level environment for overriding if we decide later.